README.rst

```diff
@@ -74,6 +74,29 @@ A Glance:
     >>> article.summary
     'The study shows that 93% of people ...'
 
+
+Keyword extraction with YAKE! (Yet Another keyword extractor)
+To learn more about YAKE! please check the official repository (https://github.com/LIAAD/yake).
+note: The lower the score, the more relevant the keyword is.
```

Comment:
curious, what does this score refer to?

Reply:
part of me feels we should score=(1-score) so bigger is better. otherwise users of this API might get confused and try to sort(..) incorrectly.

you can also rename the api call `yake` to `yake_keywords_ranked(..)` so that it is clearer.

Reply:
The score refers to the relevance of the word in text.

Like says the YAKE! documentation [(),](https://github.com/LIAAD/yake#output) the lowest score means that word are more relevant.

i am not sure if i understand, you want me to change the function name to yake_keywords_ranked in file newspaper/article.py ?